internal/keepers/service.go

```diff
@@ -284,7 +284,7 @@ func (s *onDemandUpkeepService) parallelCheck(ctx context.Context, keys []types.
 
 	// Create batches from the given keys.
 	// Max keyBatchSize items in the batch.
-	keysBatches := createBatches(keysToSend, keyBatchSize)
+	keysBatches := createBatches(keysToSend, maxWorkersBatchSize)
```

Comment:
Looks like it still isn't coming from the config.

Reply:
It should not come from the config. This is the batch of the worker, but we are talking about this one: https://github.com/smartcontractkit/ocr2keepers/blob/main/internal/keepers/ocr.go#L126-L145